services/bankid-api/serverless.yml

```diff
@@ -68,25 +68,29 @@ functions:
       - http:
           path: auth/bankid/auth
           method: post
+          private: true
           cors: true
   sign:
     handler: lambdas/sign.main
     events:
       - http:
           path: auth/bankid/sign
           method: post
+          private: true
           cors: true
   collect:
     handler: lambdas/collect.main
     events:
       - http:
           path: auth/bankid/collect
           method: post
+          private: true
           cors: true
   cancel:
     handler: lambdas/cancel.main
     events:
       - http:
           path: auth/bankid/cancel
           method: post
```

Comment:
Is there a reason not to have the parameter authorizer defined?
See cases-api for reference.

Reply:
I'm unsure whether or not it was needed. Seeing as the task only was for API Keys, and that we need to use the bankID to login, I opted to not include it.